src/ServiceInsight/Properties/Settings.settings

```diff
@@ -1,7 +0,0 @@
-﻿<?xml version='1.0' encoding='utf-8'?>
-<SettingsFile xmlns="uri:settings" CurrentProfile="(Default)">
-  <Profiles>
-    <Profile Name="(Default)" />
-  </Profiles>
-  <Settings />
-</SettingsFile>
```

Comment:
This file (and Settings.Designer.cs and the related elements in the csproj) should be untouched by this PR. If they are redundant then we can remove them in another PR (or at least, as a separate clean up commit in this PR).

Reply:
Brought them back. There are a few other issues I discovered along the way. I'll submit a refactoring PR to get them fixed.